bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "legislative-simulation",
+  "version": "0.0.0",
+  "homepage": "https://github.com/jtoney1/legislative-simulation",
+  "authors": [
+    "james <jatoney@hotmail.com>"
+  ],
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```

Comment:
I'm fine pulling them in with bower... I just figured it was less resource hungry this way.